pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -63,6 +62,9 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) (*apps.Deployment, error)
 			PriorityClassName:  cephv1.GetMgrPriorityClassName(c.spec.PriorityClassNames),
 		},
 	}
+	if c.spec.Mgr.Count > 1 {
```

Comment:
```suggestion
	if c.spec.Mgr.Count == 2 {
```

It can only be 1 or 2, right?

Reply:
yes, however if we ever allow 3 in the future, the `> 1` condition would still be valid so I'd rather leave it more general